app/websocket/app/main/routes.py

```diff
@@ -66,9 +70,15 @@ def model_inference(model_id):
 @main.route("/serve-model/", methods=["POST"])
 def serve_model():
     encoding = request.form["encoding"]
-    serialized_model = request.form["model"].encode(encoding)
     model_id = request.form["model_id"]
 
+    if request.files:
```

Comment:
add a comment describing why this check is needed.

Reply:
Done! (b017e19)